### DISCUSSION TITLE
Welcome to vue-next-select Discussions!

### DISCUSSION THREAD

**Discussion by soc221b** [soc221b/vue-next-select#138] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SupportG** [comment]
Hi. I'm find problem with computed property https://codesandbox.io/s/friendly-jones-87hus?file=/src/App.vue
Second select not update options from computed data

**Reply by soc221b** [reply]
Thanks you for the feedback, I create an issue to track this bug.
https://github.com/iendeavor/vue-next-select/issues/207
